feat: writing log file

victorltd opened the pull request:
We need to create other task,  with the scope of integration with ttc values, when its finished, getting the real conditions for trigger the actuactors. 
But it believe the scope of #23 , which is create log file, i**s concluded for now.**

pcgomesp commented:
The files seem to be consistent and comply with the issue set up, and as the tests have already been passed, I think it's ready to merge. 

But I have a question: following the proposed architecture, the function for writing to the file would have to be called at each iteration of the main actuator thread loop. However, it is only being called when the condition in line 77 is met. Shouldn't it be called every time?

RafaelBNN reviewed (changes requested):
Files very well organized.

My only suggestion would be to translate error messages and comments to English or remove the comments completely.

Also, I think it is a common practice not to commit log files to the repository. Should we add it to the .gitignore?

victorltd commented:
> Files very well organized.
> 
> My only suggestion would be to translate error messages and comments to English or remove the comments completely.
> 
> Also, I think it is a common practice not to commit log files to the repository. Should we add it to the .gitignore?


good sugestions Rafael, Both of them i changed at branch. 
Changed to the gitignore, for won't send anymore the log files, as the good practices recommend.

victorltd commented:
> The files seem to be consistent and comply with the issue set up, and as the tests have already been passed, I think it's ready to merge.
> 
> But I have a question: following the proposed architecture, the function for writing to the file would have to be called at each iteration of the main actuator thread loop. However, it is only being called when the condition in line 77 is met. Shouldn't it be called every time?

Yes Paulo, i made a mistake when i put this condition, i think i could use this value in this condition for test the function , while the real statements coming from the ttc module was built. 

But now, i fixed the mistake at this line.  Thanks

pcgomesp commented:
With all the details sorted out, I think it's safe to merge, considering that I had already tested it previously. 

Closes #23
